kylin/pom.xml

```diff
@@ -55,6 +55,11 @@
             <artifactId>commons-codec</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>20170516</version>
```

Comment:
is this a new dependency? if so, it should be added to the LICENSE file

Reply:
zeppelin-interpreter has dependecy Gson

Reply:
gson yes, and this is org.json? I didn't see it in the LICENSE file

Reply:
i mean need to use Gson instead of org.json

Reply:
Thanks very much @tinkoff-dwh and @felixcheung for your comments.  I was aware that zeppelin already has the dependency of gson, so is it mandatory to use gson instead of json?

Reply:
not necessarily. each interpreter can use its own set of things. but new binary dependencies we need to list in LICENSE file for the release.

Reply:
@felixcheung I have replaced json with gson and I will push the code soon.